p2p/enode/urlv4_test.go

```diff
@@ -55,39 +54,39 @@ var parseNodeTests = []struct {
 	},
 	{
 		rawurl: "enode://1dd9d65c4552b5eb43d5ad55a2ee3f56c6cbc1c64a5c8d659f51fcd51bace24351232b8d7821617d2b29b54b81cdefb9b3e9c37d7fd5f63270bcc9e1a6f6a439@127.0.0.1:52150",
-		wantResult: NewV4(
+		wantResult: NewV4Hostname(
 			hexPubkey("1dd9d65c4552b5eb43d5ad55a2ee3f56c6cbc1c64a5c8d659f51fcd51bace24351232b8d7821617d2b29b54b81cdefb9b3e9c37d7fd5f63270bcc9e1a6f6a439"),
-			net.IP{0x7f, 0x0, 0x0, 0x1},
+			"127.0.0.1",
 			52150,
 			52150,
 			0,
 		),
```

Comment:
Should we keep the `NewV4` test and just add `NewV4Hostname` tests? Could even be in another file like `p2p/enode/urlv4_quorum_test.go`.

Reply:
That sounds like a good idea.

Reply:
Hm, then again, since the tests make use of `parseComplete` which has been modified to use `NewV4Hostname`, the original tests don't make sense. We could duplicate the entire functionality of `parseComplete` and all its callsites, but that seems like a lot of effort for little gain.